CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.1.3
 
+- Added `shortestPath` and `shortestPaths` functions.
+
```

Comment:
[nit] no extra space - why do we need the extra `<p>` for every bullet? It makes it not match the markup for single element lists...

Reply:
ack